modules/react/_examples/stories/examples/layout/CustomColumnWidth.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+
+import {Grid, Box} from '@workday/canvas-kit-react/layout';
+import {BodyText, Heading} from '@workday/canvas-kit-react/text';
+
+export const CustomColumnWidth = () => (
+  <>
+    <Heading size="medium">Custom Column Width</Heading>
+    <Grid gridTemplateColumns="4fr 2fr 6fr" gridGap="m">
+      <Box height="120px" backgroundColor="blueberry400">
+        <BodyText size="small" textAlign="center" color="frenchVanilla100">
+          4 column width
+        </BodyText>
+      </Box>
+      <Box height="120px" backgroundColor="blueberry400">
+        <BodyText size="small" textAlign="center" color="frenchVanilla100">
+          4 column width
+        </BodyText>
+      </Box>
+      <Box height="120px" backgroundColor="blueberry500">
```

Comment:
Something looks off for the `Custom Column Width` example in the preview build:

<img width="1064" alt="image" src="https://user-images.githubusercontent.com/146020/235266713-f4efde09-3c4c-4839-ba5d-64091548e425.png">

I think there's an extra `<Box>` which shouldn't be here:

```suggestion
      </Box>
      <Box height="120px" backgroundColor="blueberry500">
```

Reply:
yeah, it was box duplication there. Thanks for catching it